pkg/gadgets/run/tracer/run.go

```diff
@@ -41,6 +44,7 @@ import (
 const (
 	ProgramContent  = "prog"
 	ParamDefinition = "definition"
+	OCIImage        = "oci-image"
```

Comment:
Also forgot to add the `Param` prefix.

I  think we should have the prefixes here to know that these are constants for the parameters